fix a compile Problem on >=vdr-1.3.47

taken from http://vdr-portal.de/board/thread.php?postid=453463#post453463
Joerg Bornkessel <hd_brummy@gentoo.org> (04 May 2006)

--- a/PLUGINS/src/burn/burn.c Sun Apr  9 17:41:44 2006
+++ b/PLUGINS/src/burn/burn.c     Mon Apr 17 19:54:12 2006
@@ -34,7 +34,7 @@
     virtual bool Start(void);
     virtual void Stop(void);
     virtual void Housekeeping(void);
-    virtual bool Active(void);
+    virtual cString Active(void);
     virtual const char *MainMenuEntry(void) { return tr(MAINMENUENTRY); }
     virtual cOsdObject *MainMenuAction(void);
     virtual cMenuSetupPage *SetupMenu(void);
@@ -196,9 +196,14 @@
     cBurnMenuBurn::Housekeeping();
 }
 
-bool cPluginBurn::Active(void)
+cString cPluginBurn::Active(void)
 {
-    return cBurnMenuBurn::JobActive();
+       if(cBurnMenuBurn::JobActive())
+    {
+               return tr("Burn job aktiv");
+    }
+    return NULL;
+
 }
 
 cOsdObject *cPluginBurn::MainMenuAction(void)
